eslint/index.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+  rules: {
+    'proptype-definition': {
```

Comment:
Let's call this `proptype-definition-above-fn`

Reply:
Sure.